clients/admin-ui/src/features/messaging-templates/messaging-templates.slice.ts

```diff
@@ -31,12 +90,79 @@ const messagingTemplatesApi = baseApi.injectEndpoints({
         method: "PUT",
         body: templates,
       }),
-      invalidatesTags: () => ["Messaging Templates"],
+      invalidatesTags: () => ["Property-Specific Messaging Templates"],
+    }),
+    // Render data from existing template- GET by id
+    getMessagingTemplateById: build.query<MessagingTemplateResponse, string>({
+      query: (templateId: string) => ({
+        url: `/messaging/templates/${templateId}`,
+      }),
+      providesTags: () => ["Property-Specific Messaging Templates"],
+    }),
+    // Update existing template
+    updateMessagingTemplateById: build.mutation<
```

Comment:
Would prefer for this to be called putMessagingTemplateById so the difference with the PATCH version is clearer.

Reply:
Done